AppCenterAuth/AppCenterAuthTests/MSAuthTests.m

```diff
@@ -1197,7 +1200,7 @@ - (void)testRefreshNeededTriggersRefresh {
   OCMStub([authMock configAuthenticationClient]).andDo(^(NSInvocation *__unused invocation) {
     self.sut.clientApplication = self.clientApplicationMock;
   });
-  id accountMock = OCMClassMock([MSALAccount class]);
+  id accountMock = OCMClassMock(MSALAccount.class);
```

Comment:
Any reason of changing this one?

Reply:
Just for consistency as I used the class property for this class mock in other tests.